lib/mpl_toolkits/axes_grid1/colorbar.py

```diff
@@ -715,6 +715,33 @@ def add_lines(self, CS):
         #tlinewidths = [col.get_linewidth()[0] for lw in CS.collections]
         ColorbarBase.add_lines(self, CS.levels, tcolors, tlinewidths)
 
+    def update_normal(self, mappable):
+        """
```

Comment:
I guess this needs some sort of test while we are here....

Reply:
Given that axes_grid1.colorbar is already deprecated, I would rather not bother...

Reply:
Oh I didn't catch that.